src/server.cpp

```diff
@@ -1881,6 +1881,20 @@ void Server::SendSetSky(u16 peer_id, const video::SColor &bgcolor,
 	Send(&pkt);
 }
 
+void Server::SendSetClouds(u16 peer_id, const float density,
+		const video::SColor &color_bright,
+		const video::SColor &color_ambient,
+		const float height,
+		const float thickness,
+		const v2f speed)
+{
+	NetworkPacket pkt(TOCLIENT_SET_CLOUDS, 0, peer_id);
+	pkt << (u16) (density  * 65535) << color_bright << color_ambient
```

Comment:
Oh, just transmitting a number between 0 and 1 in two bytes – I didn't assume we needed more than 16 bits for the density.